src/status_im/contact/subs.cljs

```diff
@@ -147,19 +146,10 @@
                :else
                (identicon/identicon chat-id)))))
 
-(defn- address= [{:keys [address] :as contact} s]
```

Comment:
IMO function's name implies that we are going to compare addresses, so I would rather avoid passing the whole contact as a parameter here. `[address1 address2]` or something `[x y]` would fit better.